requirements-test.txt

```diff
@@ -1,4 +1,6 @@
 psycopg2
-coveralls
 django-filter>=2.0
 contexttimer
+# style check for commits
+openwisp-utils[qa]
+djangorestframework
```

Comment:
redundant with https://github.com/openwisp/django-rest-framework-gis/blob/master/setup.py#L40, please remove it

Reply:
qa-checks are called before setup.py and is failing in the build